examples/nanocoap_tool/Makefile

```diff
@@ -0,0 +1,47 @@
+# name of your application
+APPLICATION = nanocoap_tool
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Include packages that pull up and auto-init the link layer.
+USEMODULE += netdev_default
+USEMODULE += auto_init_gnrc_netif
+# Specify the mandatory networking modules for IPv6
+USEMODULE += gnrc_ipv6_default
+# Additional networking modules that can be dropped if not needed
+USEMODULE += gnrc_icmpv6_echo
+
+USEMODULE += auto_init_sock_dns    # add fall-back DNS server
+USEMODULE += sock_dns              # include DNS client
+USEMODULE += gnrc_ipv6_nib_dns     # include RDNSS option handling
+
+USEMODULE += nanocoap_sock
+USEMODULE += nanocoap_vfs
+
+USEMODULE += vfs_default
+
+USEMODULE += shell
+USEMODULE += shell_cmds_default
+
+# export the whole fs to RIOT
```

Comment:
Why? Wouldn't be "./" be sufficient for an example?

Reply:
Paths in RIOT are always absolute, we don't have the concept of a current directory.
So to access `main.c` in the directory of the example we'd have to write `/main.c` which does not match the host file system.